hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/contract/AbstractContractUnbufferTest.java

```diff
@@ -54,6 +54,7 @@ public void setup() throws Exception {
   public void testUnbufferAfterRead() throws IOException {
     describe("unbuffer a file after a single read");
     try (FSDataInputStream stream = getFileSystem().open(file)) {
+      assertTrue(stream.hasCapability("in:unbuffer"));
```

Comment:
`AbstractContractUnbufferTest` was copied from Hadoop when Ozone was still on 3.2.  I forgot to remove it  when upgrading to Hadoop 3.3.  This patch reminds me to do so.

Therefore I think we should add this assertion in `hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java` instead.

If you don't feel like updating the patch, I can make the move when removing this test class.

Reply:
ok good to know. i'll update the PR soon.